kubernetes/ansible/roles/lms-ft/templates/lms-ft.env.j2

```diff
@@ -0,0 +1,56 @@
+sunbird_username=ntptest104
```

Comment:
can you remove these dummy values and variablize them?

Reply:
where I can set those variables??